Code/GraphMol/MolDraw2D/DrawTextQt.h

```diff
@@ -0,0 +1,44 @@
+//
+//  Copyright (C) 2020 Greg Landrum and T5 Informatics GmbH
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#ifndef RDKIT_DRAWTEXTQT_H
+#define RDKIT_DRAWTEXTQT_H
+
+#include <GraphMol/MolDraw2D/DrawText.h>
+
+class QPainter;
+
+namespace RDKit {
+
+// ****************************************************************************
+class DrawTextQt : public DrawText {
+ public:
+  DrawTextQt(double max_fnt_sz, double min_fnt_sz, QPainter &qp);
+
+#if 0
+  void getStringSize(const std::string &label, double &label_width,
+                     double &label_height) const override;
+#endif
+  void drawChar(char c, const Point2D &cds) override;
```

Comment:
What happens to non-ascii characters? Would they be rendered as individual utf-8 bytes?

I ask bc I know I've seen people talk in the RDK mailing list about, e.g. "±" in images.

I wonder if it would make sense to have this interface use QChar/QString.

Reply:
The API for this is set at the MolDraw2D level and it is really only setup for text made up of single-byte characters.
We could think about moving to supporting wide chars, particularly now that the freetype support is there, but that's a bigger story